src/Resolve/graphtype.jl

```diff
@@ -274,10 +274,10 @@ mutable struct Graph
             uuid = pkgs[p0]
             vnmap = get(Dict, compat[pkgs[p0]], vn)
             for (name,vs) in vnmap
-                haskey(n2u, name) || error("Unknown package $name found in the compatibility requirements of $(pkgID(pkgs[p0], uuid_to_name))")
+                haskey(n2u, name) || error("Unknown package $name found in the compatibility requirements of $(logstr_pkgID(pkgs[p0], uuid_to_name))")
```

Comment:
This isn't a log message this is an error so probably should not change it 
```suggestion
                haskey(n2u, name) || error("Unknown package $name found in the compatibility requirements of $(pkgID(pkgs[p0], uuid_to_name))")
```